src/rng/Halton_Sequence.cc

```diff
@@ -974,7 +974,8 @@ unsigned prime[] =
      99859, 99871, 99877, 99881, 99901, 99907, 99923, 99929, 99961, 99971,
      99989, 99991};
 //! Number of primes known to the Halton Sequence algorithm.
-unsigned const NUMBER_OF_KNOWN_PRIMES = sizeof(prime) / sizeof(prime[0]);
+Remember(unsigned const NUMBER_OF_KNOWN_PRIMES = sizeof(prime) /
+                                                 sizeof(prime[0]));
```

Comment:
* gcc is issuing a warning about an extra `;`.  Can you move the semicolon inside of the last closing paren?

```
source/src/rng/Halton_Sequence.cc:978:67: warning: extra ‘;’ [-Wpedantic]
                                                  sizeof(prime[0]));
                                                                   ^
```

Reply:
@KineticTheory Oh interesting - thank you for catching this.

Reply:
Moving the semicolon in seems to cause an issue with clang formatting.  In the recent commit, the `Remember` has been taken off and the ctor `Require` has been upgraded to an `Insist`.